rn_version.rb

```
@@ -0,0 +1,28 @@
+        else
+            nil
+        end
+
+    unless rn_pkg_json
+        warn "🦄 Unistyles: React Native not found. Frameworks :static will use all dependencies which might fail for older versions of React Native."
+        return nil
+    end
+
+    rn_pkg = JSON.parse(File.read(rn_pkg_json))
+    rn_version = rn_pkg['version']
+    parsed_version = rn_version.split('.')[1].to_i
```

RuboCop (1.75.5)

[convention] Redundant assignment before returning detected.

(Style/RedundantAssignment)